.dev/tests/phpunit/includes/test-coblocks-block-assets.php

```diff
@@ -98,7 +98,31 @@ public function test_editor_assets_scripts() {
 		$this->go_to( '/wp-admin/post-new.php' );
 		$this->coblocks_block_assets->editor_assets();
 
-		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered ) );
+		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered,  ) );
+
```

Comment:
This code below wasn't actually working for me. The `$path` var was incorrect so `$files` was just returning an empty array and the assertions in the foreach loop weren't running.

I've added some suggested code changes below with the changes that got it working.